lib/ibanity/api/base_resource.rb

```diff
@@ -92,7 +92,7 @@ def prepare_attributes(raw)
       attributes           = raw["attributes"]    || {}
       meta                 = raw["meta"]          || {}
       params               = base.merge(attributes).merge(meta)
-      Ibanity::Util.underscorize_hash(params)
+      Ibanity::Util.underscorize(params)
```

Comment:
Does the output message of the lib output as snake_case to the API ?

Reply:
As far as I know the payload is sent in snake case.
It seems the server transforms all payloads in snake case as well, so it doesn't matter if it's camel case or snake case.

Reply:
ok. No change required then.